tests/tests.qbs

```diff
@@ -0,0 +1,100 @@
+import qbs
+import qbs.FileInfo
+import qbs.File
+
+Project {
+    name: "QSqlMigrator.tests"
+
+
+    Application {
+        name: "ApiTest"
+        consoleApplication: true
+        type: base.concat("autotest")
+        install: true
+
+        Depends { name: "QSqlMigrator" }
+        Depends { name: "cpp" }
+        Depends { name: "Qt"; submodules: ["core", "sql", "test"] }
+        Depends { name: "SqliteMigrator" }
+
+        Group {
+            name: "api"
+            prefix: "api/"
+            files: [
+                "tst_ApiTest.cpp",
+                "M20132201_180943_CreateCars.cpp",
+                "M20132201_175827_CreateAddresses.cpp",
+                "M20131501_191807_CreateUsers.cpp",
+                "M20133001_164323_AddUsers.cpp",
+                "SqliteAddUser.cpp",
+                "M20132201_180943_CreateCars.h",
+                "M20132201_175827_CreateAddresses.h",
+                "M20131501_191807_CreateUsers.h",
+                "M20133001_164323_AddUsers.h",
+                "SqliteAddUser.h",
+                "ApiConfig.h",
+                "ApiConfig.h.example",
```

Comment:
It would be great to sort the files lists by alphabet. This helps to reduce conflicts and also moves matching headers and cpp files next to each other.